lib/pbench/server/cache_manager.py

```diff
@@ -506,7 +507,7 @@ def get_info(self, path: Path) -> JSONOBJECT:
         return fd_info
 
     @staticmethod
-    def extract(tarball_path: Path, path: Path) -> Inventory:
+    def extract(tarball_path: Path, path: str) -> Inventory:
```

Comment:
I'd debated this change earlier, and decided to keep the str-ification here and let callers use `Path` "math" without having to remember to convert. And, technically, (and despite the simplistic type hint) that made this a "PathLikeObject" (a type that doesn't actually exist and is awkward to construct), which I rather liked. I'm not sure how I feel about the change...